engine/src/test/scala/cromwell/engine/workflow/lifecycle/execution/ejea/EngineJobExecutionActorSpecUtil.scala

```diff
@@ -26,12 +26,6 @@ private[ejea] trait CanExpectCacheWrites extends Eventually { self: EngineJobExe
   def expectCacheWrite(expectedResponse: JobSucceededResponse, expectedCallCacheHashes: CallCacheHashes): Unit = {
     eventually { ejea.stateName should be(UpdatingCallCache) }
     ejea.stateData should be(SucceededResponseData(expectedResponse, Some(Success(expectedCallCacheHashes))))
-    helper.callCacheWriteActorCreations match {
-      case GotOne(creation) =>
```

Comment:
I'm not 100% sure but I wouldn't be surprised if `GotOne` was only ever used here